src/components/Text/index.tsx

```diff
@@ -13,18 +13,21 @@ interface Props {
   leading?: SpaceScale;
   display?: boolean;
   label?: boolean;
+  link?: boolean;
```

Comment:
Is the sole purpose of this prop to set styles? If so I'd suggest instead to leverage the already existing `as` prop.

Reply:
Yeah it is. That makes a lot more sense